### DISCUSSION TITLE
PikPak Is working. But videos can't be play.

### DISCUSSION THREAD

**Discussion by thuvaragan-g** [AlistGo/alist#6749] [Q&A]
PikPak Is working. It shows the folders and files. But videos can't be play. v3.35.0

**Comment by thuvaragan-g** [comment]
<img width="1016" alt="Screenshot 2024-07-13 at 10 44 10" src="https://github.com/user-attachments/assets/97652c48-e2ba-4767-9fcb-c68bbe101520">

**Comment by skyfish2023** [comment]
获取到的直链直接在浏览器中下载提示404

**Reply by thuvaragan-g** [reply]
Yes, It's same for me also.
<img width="782" alt="Screenshot 2024-07-13 at 16 54 41" src="https://github.com/user-attachments/assets/b3ab9d31-c63e-42d0-a8fa-63315576b5a0">

**Comment by anwen-anyi** [comment]
https://github.com/alist-org/alist/issues/6752

**Reply by thuvaragan-g** [reply]
Thanks @anwen-anyi.
